__tests__/frontend-proxy.ts

```diff
@@ -86,6 +86,28 @@ describe('handleRequest()', () => {
     })
   })
 
+  describe('returned response set cookie to 0 on mobile browser', () => {
+    //TODO: Backend Legacy fails if I set user-agent to 0 // Backend.Legacy
+    test.each([Backend.Frontend])('%p', async (backend) => {
```

Comment:
It's better to have a each for different user agents here.

Reply:
We know the "detection" is okay, but far from great. Changes to the implementation might improve the result but actually break some examples on the way.
Would you add tests for different user-agent strings here?